lib/ical/event.js

```diff
@@ -24,7 +24,7 @@ ICAL.Event = (function() {
    * @param {Boolean} options.strictExceptions
    *          When true, will verify exceptions are related by their UUID
    * @param {Array<ICAL.Component|ICAL.Event>} options.exceptions
-   *          Exceptions to this event, either as components or events
+   *          Exceptions to this event, either as components or events. If not specified exceptions will automatically be set in relation of component's parent
```

Comment:
Can you wrap this to 80 characters, indenting the next lines by two additional spaces?